controllers/submariner/submariner_controller.go

```diff
@@ -306,10 +307,11 @@ func (r *SubmarinerReconciler) reconcileEngineDaemonSet(instance *submopv1a1.Sub
 }
 
 func (r *SubmarinerReconciler) reconcileNetworkPluginSyncerDeployment(instance *submopv1a1.Submariner,
-	reqLogger logr.Logger) (*appsv1.Deployment, error) {
+	clusterNetwork *network.ClusterNetwork, reqLogger logr.Logger) (*appsv1.Deployment, error) {
 	// Only OVNKubernetes needs networkplugin-syncer so far
 	if instance.Status.NetworkPlugin == "OVNKubernetes" {
```

Comment:
Can we create a constant for OVNKubernetes and use it here as well as pkg/discovery/network/ovnkubernetes.go file

Reply:
oh let me handle that while I'm at it...

Reply:
done